include/nv/target

```diff
@@ -7,12 +7,26 @@
 //
 //===----------------------------------------------------------------------===//
 
+// NOTICE:
+//  This header contains a preview of a portability system that enables
+//  CUDA accelerated development for a set of several compilers. The interfaces
```

Comment:
Say "CUDA C++ development with NVC++, NVCC, and supported host compilers". We don't want to suggest other CUDA compilers exist.